professional_python_exercises_2_githubcli/cli/root.py

```diff
@@ -0,0 +1,9 @@
+import typer
+from .stars import stars_cmd
+from .status import status_cmd
+from .user import user_cmd
+
+app = typer.Typer(no_args_is_help=True)
```

Comment:
Again, I really like that you can expand the solution in an easy way with this structure. I suppose it is rather common to have a small main root file.